src/Microsoft.VisualBasic/pkg/Microsoft.VisualBasic.builds

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <!-- Temporary workaround, VisualBasic projects cannot be built on non-Windows platforms. See issue #5230. -->
```

Comment:
If we are able to build VB project now on non-Windows, then please remove this comment too.

@weshaggard PTAL as well.